apps/backend/src/mutations/FapMutations.ts

```diff
@@ -381,6 +382,123 @@ export default class FapMutations {
       });
   }
 
+  /**
+   * Assigns all remaining reviews needed for a FAP among its members. Attempts to distribute reviews as evenly as possible.
```

Comment:
This is a good note here as the functionality is a bit complicated it is good to have an overview of what we want to actually achieve here.